lib/plugins/bed.js

```diff
@@ -4,8 +4,11 @@ module.exports = inject
 
 const CARDINAL_DIRECTIONS = ['south', 'west', 'north', 'east']
 
-function noop (err) {
-  if (err) throw err
+function callbackify (f) {
```

Comment:
Seems duplicated in a lot of place why not add it into a file and import it ?

Reply:
Well, noop was duplicated as well. And actually there's a couple places where callbackify is different because cb is not the last parameter

Reply:
For the different callbackify, it should be possible to refactor them into one by adding some extra optionals parameters like the position of the callback

Reply:
Look at digging#callbackify though, I don't think it's possible to generalize that

Reply:
I think a better solution for digging is just to not use a callbackify function and do
```js
  const callbackDig = util.callbackify(dig)
  bot.dig = (block, forceLook, cb) => {
    if (typeof forceLook === 'function') {
      cb = forceLook
      forceLook = true
    }
    cb = cb || noop
    callbackDig(block, forceLook, cb)
  }
```

Reply:
I will look at this later

Reply:
For now I did already extract the functions into a module, I have left the two exceptions as-is